src/runner.js

```diff
@@ -3,14 +3,25 @@ import { remote } from 'webdriverio'
 import { getMetricParams, getThrottleNetworkParam, getThrottleCpuParam } from './utils'
 
 export default async function runPerformanceTest (username, accessKey, argv, name, build, logDir) {
-    const { site, platformName: platform, browserVersion: version, tunnelIdentifier, parentTunnel } = argv
+    const { site, platformName, browserVersion, tunnelIdentifier, parentTunnel } = argv
     const metrics = getMetricParams(argv)
     const networkCondition = getThrottleNetworkParam(argv)
     const cpuRate = getThrottleCpuParam(argv)
     const sauceOptions = {
-        name, build, extendedDebugging: true, capturePerformance: true,
-        ...(tunnelIdentifier ? { tunnelIdentifier } : {}),
-        ...(parentTunnel ? { parentTunnel } : {})
+        'sauce:options': {
+            name,
+            build,
+            extendedDebugging: true,
+            capturePerformance: true,
+            crmuxdriverVersion: 'beta',
```

Comment:
Can we select this through argv? (We can do it as a separate PR as well)

Reply:
This is temporary. Once we released the our internal driver as stable I make a new PR removing this capability or make it configurable via parameter.